packages/blade/src/components/NativeAria/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from './NativeAria.native';
+export * from './NativeAria.native';
```

Comment:
Why is this exporting `.native` files? It should just do this
```suggestion
export { default } from './NativeAria';
export * from './NativeAria';
```